.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: node_js
+node_js: "8"
+stages: 
+    - name: unit_tests
+    
+jobs:
+    include:
+        - stage: unit_tests
+          env: CHECK=UNIT_TESTS
+          script: 
+            - "npm install mocha chai"
```

Comment:
i would recommend to change this to just `npm install`, instead of specifying the packages individually.

Reply:
It has been changed to npm install to cover all dependencies.